src/components/calcite-pick-list/readme.md

```diff
@@ -1,5 +1,7 @@
 # calcite-pick-list
 
+The `calcite-pick-list` component contains `calcite-pick-list-items` stacked in a panel that can be selected or deselected via shift click to be shown on the view.
```

Comment:
The wording here sounds like you have to use shift click. Perhaps add a period after `deselected` and note that shift click works for selecting multiple items at a time (deselecting does not work with shift-click).

Reply:
It's probably also worth calling out the change event that gets fired whenever a selection changed that provides the full list of selections in a JavaScript Map.